src/sync/pull-request.ts

```diff
@@ -57,18 +61,25 @@ export default async function(
 	const asyncTags:any[] = [];
 	// TODO: use graphql here instead of rest API
 	await statsd.asyncTimer(
+		// Retry up to 6 times pausing for 10s, for *very* large repos we need to wait a while for the result to succeed in dotcom
 		async () => {
-			({
-				data: edges,
-				status,
-				headers
-			} = await github.pulls.list({
-				...vars,
-				state: "all",
-				sort: "created",
-				direction: "desc"
-				// Retry up to 6 times pausing for 10s, for *very* large repos we need to wait a while for the result to succeed in dotcom
-			}));
+			(
+
+				{
+					data: edges,
+					status,
+					headers
+				} = useNewGHClient?
+					await github.pulls.list({ ...vars, state: "all", sort: "created", direction: "desc"})
+					:
+					await newGithub
+						.getPullRequests(repository.owner.login, repository.name, perPage, Number(cursor),
+							{
+								...vars,
```

Comment:
the `...vars` is passing the `owner` and `repo` options into the function that are already defined as specific parameters. We shouldn't pass an untyped `options` object here, imo (see comment above).